lib/wallet/http.js

```diff
@@ -1064,6 +1064,35 @@ class HTTP extends Server {
       return res.json(200, mtx.getJSON(this.network));
     });
 
+    this.post('/wallet/:id/auction-txs', async (req, res) => {
```

Comment:
Will do for the comment ! For the name, in fact I was aiming for one single entry point for the whole process down the line, hence a generic name. In the futur, I hope to add redeem & register, just because it would be nice to just create everything at the start, ask for broadcast at good height and then forget about it.

Reply:
I think that the name of the endpoint should be `POST /wallet/:id/auction/:name`. This more closely follows the style of the other endpoints, as none of them have dashes in them. It allows for the possibility of additional transactions to be added besides bid and reveal in the future

Reply:
`POST /wallet/:id/auction/:name` I like this too

Reply:
Ok my final opinion on this is naming the endpoint `POST /wallet/:id/auction`.
I think you can keep the function name `createAuctionTxs()`

@tynes as I look through the rest of the wallet HTTP API, this matches the pattern. Adding `/:name` to queries seems to be used just in `GET` requests, and all the `POST` requests expect the name and all other parameters to be in the query JSON data.